include/sound/sof/topology.h

```diff
@@ -37,6 +37,8 @@ enum sof_comp_type {
 	/* keep FILEREAD/FILEWRITE as the last ones */
 	SOF_COMP_FILEREAD,	/**< host test based file IO */
 	SOF_COMP_FILEWRITE,	/**< host test based file IO */
+	SOF_COMP_KPB, /* A key phrase buffer component */
+	SOF_COMP_SELECTOR, /* channel selector */
```

Comment:
this isn't aligned with the note above to "keep FILEREAD/FILEWRITE as the last ones"